docs: document network access requirement for DruidConnection feature (#449)
stackabletech/superset-operator: docs/modules/superset/pages/usage-guide/connecting-druid.adoc

metadata:
  name: superset-druid-connection
spec:
  superset:  # <1>
    name: superset
    namespace: default
  druid:  # <2>
    name: druid
    namespace: default

----

> <1> The `name` and `namespace` in `spec.superset` refer to the Superset cluster that you want to connect. Following our example above, the name is `superset`.

Reviewer: Keep the wording consistent? I like the #2 better so here the proposal.
```suggestion
<1> In `spec.superset` you specify the `name` and `namespace` of your Superset cluster. Following our example above, the name is `superset`.
```
